package.json

```diff
@@ -30,14 +30,22 @@
     "prepublishOnly": "npm test"
   },
   "dependencies": {
+    "@date-io/date-fns": "^1.1.0",
+    "date-fns": "^2.0.0-alpha.27",
+    "final-form": ">=4.12.0",
+    "final-form-arrays": ">=1.1.2",
     "history": "^4.7.2",
     "js-logger": "^1.6.0",
+    "material-ui-pickers": "^2.2.4",
     "prop-types": "^15.5.7",
+    "react-final-form": ">=4.1.0",
+    "react-final-form-arrays": "2.0.1",
     "swagger-client": "3.8.22"
   },
   "peerDependencies": {
     "@material-ui/core": ">=3.9.0",
     "@material-ui/icons": ">=3.0.0",
+    "material-ui-pickers": "^2.2.4",
```

Comment:
Dependencies should be in either `peerDependencies` or `dependencies`, but not both.

Reply:
My mistake. Will move it to `depedencies` for now.